crates/sui-move-build/tests/linter/native_primitives_only.move

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
We will also want to comment these and add a copyright. 
And we should probably split this into positive/negative tests

Reply:
We still need a comment about what is being tested, and about we should still split this into positive/negative tests